scripts/elements.js

```diff
@@ -13,4 +25,46 @@ export const navbar = document.querySelector(".todo-app__header");
 export const initialTaskContainer = document.querySelector(
   ".task-container__initial-state"
 );
+export const loadMoreButton = document.getElementById("load-more");
 export const tabs = document.querySelectorAll(".filter-tab");
+
+export let searchText = "";
+
+deleteTaskInputCard.addEventListener("click", resetTaskInput);
+
+toggleButton.addEventListener("click", handleToggle);
+
+initialTaskContainer.addEventListener("click", handleToggle);
+
+addTaskButton.addEventListener("click", () => handleCreateTodo(todos));
+
+taskInputText.addEventListener("keyup", (event) => {
+  if (event.key === "Enter") {
+    handleCreateTodo(todos);
+  }
+});
+
+taskInputText.addEventListener("input", validateInput);
+
+searchButton.addEventListener("click", () => {
+  navbar.classList.toggle("show-search-box");
+});
+
+searchInput.addEventListener("keyup", async () => {
+  searchText = searchInput.value.toLowerCase().trim();
+  let searchResult = await debouncedSearchData(searchText, todos);
+  renderTodoList(searchResult);
+});
+
+loadMoreButton.addEventListener("click", handleLoadMoreTask);
+
+tabs.forEach((tab) => {
+  tab.addEventListener("click", () => {
+    const tabId = tab.id;
+    if (tabId === ALL) {
+      filterTasksByStatus(tabId);
+    } else if (tabId === COMPLETE) {
+      filterTasksByStatus(tabId);
+    } else filterTasksByStatus(tabId);
+  });
```

Comment:
Isn't all the conditional branches doing the same thing?
```suggestion
  tab.addEventListener("click", () => filterTasksByStatus(tab.id));
```

Reply:
> Isn't all the conditional branches doing the same thing?

Yeah. But I did this based on [https://github.com/njm-wtag/Todo_Vanilla_JS/pull/9#discussion_r1433544878](https://github.com/njm-wtag/Todo_Vanilla_JS/pull/9#discussion_r1433544878) review. Maybe I didn't get it properly.